modules/tile-converter/docs/api-reference/3d-tiles-converter.md

```diff
@@ -4,7 +4,7 @@
   <img src="https://img.shields.io/badge/From-v3.0-blue.svg?style=flat-square" alt="From-v3.0" />
 </p>
 
-The `Tiles3DConverter` class converts an 3D Tiles tileset. It converts between the OGC 3D Tiles and the I3S formats.
+The `Tiles3DConverter` class converts an I3S layer. It converts between the I3S formats and the OGC 3D Tiles.
```

Comment:
Bot formats are OGC, mention in both places or only one. The way I wrote it OGC covered both.